frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/mirroring-card/mirroring-card.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+
+import { ocsResource } from '../../../../resources';
+import '../../common/storage-efficiency/storage-efficiency-card.scss';
+
+export const MirroringItemBody: React.FC<MirroringItemBodyProps> = React.memo(
+  ({ title, isLoading, error, children }) => {
+    const { t } = useTranslation();
+
+    let status: React.ReactElement;
+
+    if (isLoading) {
+      status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
+    } else if (error) {
+      status = (
+        <span className="co-dashboard-text--small text-muted">
+          {t('ceph-storage-plugin~Not available')}
+        </span>
+      );
+    } else {
+      status = <span className="ceph-storage-efficiency-card__item-text">{children}</span>;
+    }
+    return (
+      <div className="co-inventory-card__item">
+        <div className="ceph-storage-efficiency-card__item-title">{title}</div>
+        <div
+          className="ceph-storage-efficiency-card__item-status"
+          data-test={`${title}-mirroring-card-status`}
+        >
+          {status}
+        </div>
+      </div>
+    );
+  },
+);
+
+const MirroringCard: React.FC<DashboardItemProps> = ({
+  watchK8sResource,
+  stopWatchK8sResource,
+  resources,
+}) => {
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    watchK8sResource(ocsResource);
+    return () => {
+      stopWatchK8sResource(ocsResource);
+    };
+  }, [watchK8sResource, stopWatchK8sResource]);
+
+  const ocs = resources?.ocs;
+  const mirroringStatus: boolean = ocs?.data?.[0]?.spec.hasOwnProperty('mirroring');
```

Comment:
```suggestion
  const ocsData = ocs?.data as K8sResourceKind[];
  const cluster = ocsData?.find((item: StorageClusterKind) => item.status.phase !== 'Ignored');
  const mirroringStatus: boolean = cluster?.spec?.mirroring ?  cluster?.spec?.mirroring : false;
```

is this works?

Reply:
That should work too... will update the code...